src/tests/illumidesk/authenticators/test_lti13_authenticator.py

```diff
@@ -134,8 +133,7 @@ async def test_authenticator_returns_username_in_auth_state_with_with_given_name
     ):
         with patch.object(LTI13LaunchValidator, 'validate_launch_request', return_value=True):
             result = await authenticator.authenticate(request_handler, None)
-            monkeypatch.setitem(auth_state_dict, 'name', 'foobar')
-
+            auth_state_dict['name'] = 'Foo Bar'
             assert result['name'] == auth_state_dict['name']
```

Comment:
the same, can we change it to have a fixed expected value?

Reply:
Yes. I guess I got carried away removing the unnecessary monkeypatches.